### PR TITLE
Refactor CSS

### PR DESCRIPTION
**Pull request by dfosco**
- [x] Add BEM syntax (to avoid conflicts in the future)
 - [x] Organize utils folder
 - [x] Import utils folder into every stylesheet
 - [x] Normalize util class usage across all styles
 - [x] Install `vtex-css-utils`
 - [x] Isolates theme styles under the `.theme` wrapper class (added by the SDK to App container)

### CODE REVIEW

**Comment by tamorim** [comment]
@dfosco `This branch has conflicts`

File: src/components/CategoryHeader/ExplorerPanel/ExplorerPanel.less [hunk 1]
```diff
@@ -69,7 +69,7 @@
     }
   }
 
-  &-leave {
+  &__leave {
     opacity: 1;
     top: 0px;
```
Comment (tamorim) [inline src/components/CategoryHeader/ExplorerPanel/ExplorerPanel.less]: @dfosco the `&-enter` and `&-leave` classes should be left alone, they're being used by `ReactCSSTransitionGroup`.

Comment (dfosco) [inline src/components/CategoryHeader/ExplorerPanel/ExplorerPanel.less]: @tamorim Oh snap, `find`/`replace` got these by mistake! Should I fix them and push?

Comment (tamorim) [inline src/components/CategoryHeader/ExplorerPanel/ExplorerPanel.less]: @dfosco yeah! :D

Comment (dfosco) [inline src/components/CategoryHeader/ExplorerPanel/ExplorerPanel.less]: @tamorim I fixed the classes, but the animation is still broken — the component doesn't seem to have `ReactCSSTransitionGroup` implemented...

Comment (tamorim) [inline src/components/CategoryHeader/ExplorerPanel/ExplorerPanel.less]: @dfosco As discussed IRL, it's working :)

Comment (dfosco) [inline src/components/CategoryHeader/ExplorerPanel/ExplorerPanel.less]: @tamorim :ok_hand:

**Comment by dfosco** [comment]
@tamorim How are we going with this PR? Is it viable for merging already?

**Comment by brenoc** [comment]
@dfosco this depends on the SDK adding the classe "theme". You should make a PR there as well. Once merged, we can merge this one.

**Comment by dfosco** [comment]
@brenoc Created the PR on the SDK: https://github.com/vtex-apps/storefront-sdk/pull/57

**Pull request merged**